src/Fable.React/Fable.Helpers.React.fs

```diff
@@ -878,6 +886,40 @@ let inline ofList (els: ReactElement list): ReactElement = unbox(List.toArray el
 /// Returns an array **from .render() method**
 let inline ofArray (els: ReactElement array): ReactElement = unbox els
 
+[<RequireQualifiedAccess>]
+module ReactElementType =
+    let inline ofComponent<'comp, 'props, 'state when 'comp :> Component<'props, 'state>> : ReactComponentType<'props> =
+        unbox jsConstructor<'comp>
+
+    let inline ofFunction<'props> (f: 'props -> ReactElement) : ReactComponentType<'props> =
+        unbox f
+
+    let inline ofHtmlElement<'props> (name: string): ReactElementType<'props> =
+        unbox name
+
+    /// Create a ReactElement to be rendered from an element type, props and children
+    let inline create<'props> (comp: ReactElementType<'props>) (props: 'props) (children: ReactElement seq): ReactElement =
+        createElement(comp, props, children)
+
+type PropsEqualityComparison<'props> = 'props -> 'props -> bool
+
+/// React.memo is a higher order component. It’s similar to React.PureComponent but for function components instead of
+/// classes.
+///
+/// If your function component renders the same result given the same props, you can wrap it in a call to React.memo
+/// for a performance boost in some cases by memoizing the result. This means that React will skip rendering the
+/// component, and reuse the last rendered result.
+///
+/// By default it will only shallowly compare complex objects in the props object. If you want control over the
+/// comparison, you can also provide a custom comparison function as the second argument.
+[<Import("memo", from="react")>]
+let memo<'props> (render: 'props -> ReactElement, areEqual: PropsEqualityComparison<'props> option) : ReactComponentType<'props> =
```

Comment:
Instead of forcing users to write `memo (renderFunction, None)` maybe we could offer two different functions:

```fsharp
let memo renderFn = import "memo" "react"
let memoWith renderFn areEqual = import "memo" "react"
```